### ISSUE TITLE
Not found: Op type not registered 'SigmoidGrad'

### ISSUE THREAD

**Issue by parilo** [tensorflow/tensorflow#3944]
### Environment info
Operating System:
Mac OS X 10.11.6 (15G31)

Installed version of CUDA and cuDNN: 
no cuda

Installed from sources,
git clone https://github.com/tensorflow/tensorflow.git --branch v0.10.0rc0 --single-branch Tensorflow

1. The commit hash (`git rev-parse HEAD`)
3cb39956e622b322e43547cf2b6e337020643f21

2. The output of `bazel version`
........
Build label: 0.2.1-homebrew
Build target: bazel-out/local_darwin-fastbuild/bin/src/main/java/com/google/devtools/build/lib/bazel/BazelServer_deploy.jar
Build time: Fri Apr 1 00:35:17 2016 (1459470917)
Build timestamp: 1459470917
Build timestamp as int: 1459470917

1. create pip package and install
2. make tensorflow graph.ph file
3. make libtensorflow.so with
bazel build :libtensorflow.so
4. create c++ program and try to load this pb file
status = ReadBinaryProto(Env::Default(), dir + "/" + tfGraphFilename, &graph_def);
5. error at runtime
Not found: Op type not registered 'SigmoidGrad'

I tried tf.sigmoid and tf.nn.sigmoid, same error
tensorflow version 0.8 works fine

**Comment by yaroslavvb** [comment]
What if you use `:libtensorflow_cc.so` instead?

**Comment by parilo** [comment]
:libtensorflow_cc.so solves the problem
